suavestd/api/protocols/Bundle.mdx

```diff
@@ -0,0 +1,31 @@
+# Bundle
+
+Bundle is a library with utilities to interact with the Flashbots bundle API described in https://docs.flashbots.net/flashbots-auction/advanced/rpc-endpoint#eth_sendbundle
```

Comment:
Should wrap this as a link, i.e. "described [here](https://docs.flashbots.net/flashbots-auction/advanced/rpc-endpoint#eth_sendbundle)"